router-protobuf/src/main/kotlin/io/moia/router/proto/ProtoSerializationHandler.kt

```diff
@@ -2,19 +2,19 @@ package io.moia.router.proto
 
 import com.google.common.net.MediaType
 import com.google.protobuf.GeneratedMessageV3
-import io.moia.router.ResponseEntity
 import io.moia.router.SerializationHandler
 import java.util.Base64
 
 class ProtoSerializationHandler : SerializationHandler {
 
     private val json = MediaType.parse("application/json")
```

Comment:
I guess we could also use `org.apache.http.entity.ContentType` here

Reply:
I switched back to MediaType - see https://github.com/moia-dev/lambda-kotlin-request-router/pull/6 for details